src/stemmabench/stemma_generator.py

```diff
@@ -34,9 +36,11 @@ def __init__(
         else:
             self.config = StemmaBenchConfig.from_yaml(config_path)
         self.depth = self.config.stemma.depth
+        self.missing_manuscripts_rate = self.config.stemma.missing_manuscripts.rate
         self._levels: List[Dict[str, List[str]]] = []
         self.texts_lookup = {}
         self.edges = []
+        self.random_state = random_state
```

Comment:
Is this for testing purpose ? If so, can you add an inline comment ?

Reply:
- [x] Remove `random_state` property.